<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_profile</class>
 <widget class="QWidget" name="settings_profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_text_profile">
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>You are editing profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_active_profile">
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="cb_profile">
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save_changed_profile">
        <property name="font">
         <font>
          <family>Roboto Light</family>
         </font>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel_changed_profile">
        <property name="font">
         <font>
          <family>Roboto Light</family>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_change_profile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Roboto Light</family>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="text">
         <string>Change Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add_profile">
        <property name="font">
         <font>
          <family>Roboto Light</family>
         </font>
        </property>
        <property name="text">
         <string>Add Profile</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="layout_form">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>14</number>
     </property>
     <property name="verticalSpacing">
      <number>14</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QCheckBox" name="check_manage_node">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="text">
        <string>Manage Node</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="check_exit_close">
       <property name="font">
        <font>
         <family>Roboto Light</family>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Exit on close</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_name">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>edit_name</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="edit_name">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Name of Profile</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_host">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="text">
        <string>Host:</string>
       </property>
       <property name="buddy">
        <cstring>edit_host</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="edit_host">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="whatsThis">
        <string>IP-address or hostname of blockchain node.</string>
       </property>
       <property name="placeholderText">
        <string>localhost</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_port">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
       <property name="buddy">
        <cstring>edit_port</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="edit_port">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="placeholderText">
        <string>8374</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_rpc_user">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
       <property name="buddy">
        <cstring>edit_rpc_user</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="edit_rpc_user">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="placeholderText">
        <string>See multichain.conf</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_rpc_password">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="buddy">
        <cstring>edit_rpc_password</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="edit_rpc_password">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>see multichain.conf</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_reset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto Light</family>
           </font>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_save">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Roboto Light</family>
           </font>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="check_box_use_ssl">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="text">
        <string>Use SSL</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="check_activate_profile">
       <property name="font">
        <font>
         <family>Roboto Light</family>
        </font>
       </property>
       <property name="text">
        <string>Activate Profile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
